src/go/rpk/pkg/cli/cmd/cluster/license/info.go

```diff
@@ -53,8 +53,9 @@ func newInfoCommand(fs afero.Fs) *cobra.Command {
 						Organization string
 						Type         string
 						Expires      string
+						Checksum     int64
 						Expired      bool `json:"license_expired,omitempty"`
-					}{info.Properties.Organization, info.Properties.Type, tm, expired}, "", "  ")
+					}{info.Properties.Organization, info.Properties.Type, tm, info.Properties.Checksum, expired}, "", "  ")
```

Comment:
Is this info.Properties.Checksum going to be an error if someone uses the rpk license getter CLI on redpanda 22.2?

For the purposes of this PR, it might be simpler just not to include the checksum in the CLI output: it's not exactly meant for human eyeballs anyway.

Reply:
I just checked, won't crash:
```
{
  "Organization": "redpanda-testing",
  "Type": "enterprise",
  "Expires": "Jul 11 2122",
  "Checksum": 0
}
```

Reply:
So your saying if format == `json` include it, otherwise, hide it?